src/app/ApplicationTemplate.Shared.Views/Configuration/ViewServicesConfiguration.cs

```diff
@@ -31,6 +33,8 @@ public static IServiceCollection AddViewServices(this IServiceCollection service
 			.AddSingleton<IConnectivityProvider, ConnectivityProvider>()
 			.AddSingleton<IEmailService, EmailService>()
 			.AddSingleton<IMemoryProvider, MemoryProvider>()
+			.AddSingleton<IReviewPrompter, ReviewPrompter>()
+			.AddSingleton<IReviewService<ReviewSettings>, ReviewService.ReviewService<ReviewSettings>>()
```

Comment:
I would suggest you use a derived type of `ReviewSettings` in this sample scenario to make sure that customization works well.